ledger-core/virtual/integration/method_test.go

```diff
@@ -70,6 +70,47 @@ func Method_PrepareObject(ctx context.Context, server *utils.Server, state paylo
 	server.WaitActiveThenIdleConveyor()
 }
 
+func tolerableFlags() contract.MethodIsolation {
+	return contract.MethodIsolation{
+		Interference: contract.CallTolerable,
+		State:        contract.CallDirty,
+	}
+}
+
+func intolerableFlags() contract.MethodIsolation {
+	return contract.MethodIsolation{
+		Interference: contract.CallIntolerable,
+		State:        contract.CallDirty,
+	}
+}
+
+// for happy path tests
+func assertCalleeResponse(t *testing.T, res *payload.VCallResult, objectCaller reference.Global, flagsCaller contract.MethodIsolation) {
+	require.Equal(t, payload.CTMethod, res.CallType)
+	require.Equal(t, objectCaller, res.Caller)
+	assert.Equal(t, flagsCaller.Interference, res.CallFlags.GetInterference()) // copy from VCallRequest
+	assert.Equal(t, flagsCaller.State, res.CallFlags.GetState())
```

Comment:
CallOutgoing.pulse must be `server.GetPulse().PulseNumber` , check please

Reply:
outgoing  = server.GetPulse().PulseNumber only when send first request, how i understand it

Reply:
moved this check in test